pkg/component/load.go

```diff
@@ -9,8 +9,12 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"runtime"
+	"strings"
 
 	"github.com/elastic/go-ucfg/yaml"
+
+	yamlv3 "gopkg.in/yaml.v3"
```

Comment:
nit: we are using both `github.com/elastic/go-ucfg/yaml` and `gopkg.in/yaml.v3` at the same time

Reply:
i wanted to reduce ucfg step. and keeping ucfg one for NewConfigFrom use cases i don't want to address in this PR if that makes sense.
otherwise insteadn of unmarshal it would be
`unmarshal -> ucfg.NewFrom -> Unpack`